### PR TITLE
docs: update OCR pre-annotations notebook

### PR DESCRIPTION
**Pull request by Jonas1312**
missing link to store_with_ocr.gif. The file is added in this PR

### CODE REVIEW

File: recipes/ocr_pre_annotations.ipynb [hunk 1]
```diff
@@ -1,12 +1,23 @@
 {
```
Comment (PierreLeveau) [inline recipes/ocr_pre_annotations.ipynb:1]: <p>Pre-annotating your data with OCR will allow you to gain significant time when annotating transcriptions in Kili. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (PierreLeveau) [inline recipes/ocr_pre_annotations.ipynb:1]: <p>We will then use the Google Vision API to perform Optical Character Recognition on the different text boxes in the image. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (PierreLeveau) [inline recipes/ocr_pre_annotations.ipynb:1]: <p>We can now create the project ontology.</p><p> </p><p>For a transcription task on images, the ontology is a classification job with nested transcriptions for each category:</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (PierreLeveau) [inline recipes/ocr_pre_annotations.ipynb:1]: <p> </p><p>Let's authenticate in Kili and create our project:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (PierreLeveau) [inline recipes/ocr_pre_annotations.ipynb:1]: <p>We will now see how to perform OCR preannotation on our image using Google Vision API.</p><p> </p><p>and the itemized lists should be fixed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (PierreLeveau) [inline recipes/ocr_pre_annotations.ipynb:1]: <p> </p><p>We can now start to add OCR pre-annotations to the asset metadata:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (PierreLeveau) [inline recipes/ocr_pre_annotations.ipynb:1]: <p>should not be code-formatted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (PierreLeveau) [inline recipes/ocr_pre_annotations.ipynb:1]: <p>missing image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

File: docs/sdk/tutorials/ocr_pre_annotations.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
+<!-- FILE AUTO GENERATED BY docs/utils.py DO NOT EDIT DIRECTLY -->
```
Comment (PierreLeveau) [inline docs/sdk/tutorials/ocr_pre_annotations.md:1]: ignore this comment

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: recipes/ocr_pre_annotations.ipynb (hunk 1 above)
Comment (Jonas1312) [reply recipes/ocr_pre_annotations.ipynb:1]: bullet point lists don't show well on reviewbn

looks like this on mkdocs:

image

Comment (Jonas1312) [reply recipes/ocr_pre_annotations.ipynb:1]: same it doesn't show well on reviewnb

Comment (Jonas1312) [reply recipes/ocr_pre_annotations.ipynb:1]: image is in the PR

**Comment by Jonas1312** [comment]
@PierreLeveau  we can merge

**Pull request merged**